profiles/common/modules/features/nexteuropa_dgt_connector/tmgmt_poetry/plugins/tmgmt_poetry.plugin.inc

```diff
@@ -80,12 +80,8 @@ class TMGMTPoetryTranslatorPluginController extends TMGMTDefaultTranslatorPlugin
    * and the number of items.
    */
   public function canTranslate(TMGMTTranslator $translator, TMGMTJob $job) {
-    // Check that only one item is being requested.
-    $items = $job->getItems();
-    if (count($items) > 1) {
-      return FALSE;
-    }
     // Check that the item does not have pending requests.
+    $items = $job->getItems();
```

Comment:
Yes, it's a temporary change which will be reverted in NEPT-1117, because only than we'll have the new translator. All changes are going into a feature branch, so it won't have an impact in main branches. So, I ask for the acceptance of this temporary change here, pledging it will be reverted.